patroni/ctl.py

```diff
@@ -119,15 +120,21 @@ def auth_header(config):
         return {'Authorization': 'Basic ' + base64.b64encode(config['restapi']['auth'].encode('utf-8')).decode('utf-8')}
 
 
-def post_patroni(member, endpoint, content, headers=None):
+def request_patroni(request_type, member, endpoint, content=None, headers=None):
```

Comment:
Order of parameters looks confusing...
Either it should be `member, request_type, endpoint` or `request_type, endpoint, member` (In my opinion request_type and endpoint should be next to each other)